src/cockpit/superuser.py

```diff
@@ -146,7 +135,9 @@ def peer_authorization_request(self, peer: Peer, message: Optional[str], prompt:
     def __init__(self, router: Router, privileged: bool = False):
         super().__init__(router)
 
-        self.bridges = list(SUPERUSER_BRIDGES)
+        # name → (label, spawn, env)
+        self.superuser_rules: Dict[Optional[str], Tuple[str, List[str], Dict[str, str]]] = {}
```

Comment:
Honestly, I don't care for this — but I have some help coming in a new PR.

Reply:
OK, sorry then -- I thought annotating non-trivial data structures like this would be helpful. Please let me know how you want this changed, I'm not very proficient in this yet.